sdk/monitor/azquery/example_test.go

```diff
@@ -194,7 +194,7 @@ func ExampleMetricsClient_QueryResource() {
 			Timespan:        to.Ptr(azquery.NewTimeInterval(time.Date(2022, 12, 25, 0, 0, 0, 0, time.UTC), time.Date(2022, 12, 25, 12, 0, 0, 0, time.UTC))),
 			Interval:        to.Ptr("PT1M"),
 			MetricNames:     nil,
-			Aggregation:     to.Ptr("Average,count"),
+			Aggregation:     []*azquery.AggregationType{to.Ptr(azquery.AggregationTypeAverage), to.Ptr(azquery.AggregationTypeCount)},
```

Comment:
The `to.SliceOfPtrs()` function will make this easier.
```go
Aggregation: to.SliceOfPtrs(azquery.AggregationTypeAverage, azquery.AggregationTypeCount),
```

Reply:
That's really cool! Thanks for pointing it out.
fixed in [ffaf2f4](https://github.com/Azure/azure-sdk-for-go/pull/19981/commits/ffaf2f45111bb13afbe765ed56b34bd1fe047e96)